waku/v2/node/wakunode2_setup_metrics.nim

```diff
@@ -24,24 +25,44 @@ proc startMetricsServer*(serverIp: ValidIpAddress, serverPort: Port) =
 
     info "Metrics HTTP server started", serverIp, serverPort
 
+proc parseCollectorIntoF64(collector: Collector): float64 = 
+  var total = 0.float64
+  for key in collector.metrics.keys():
+    try:
+      total = total + collector.value(key)
+    except KeyError:
+      discard
+  return total
 
 proc startMetricsLog*() =
   # https://github.com/nim-lang/Nim/issues/17369
   var logMetrics: proc(udata: pointer) {.gcsafe, raises: [Defect].}
 
+  var cumulativeErrors = 0.float64
+  var cumulativeConns = 0.float64
+
   logMetrics = proc(udata: pointer) =
     {.gcsafe.}:
       # TODO: libp2p_pubsub_peers is not public, so we need to make this either
       # public in libp2p or do our own peer counting after all.
-      var totalMessages = 0.float64
-      for key in waku_node_messages.metrics.keys():
-        try:
-          totalMessages = totalMessages + waku_node_messages.value(key)
-        except KeyError:
-          discard
-
-    info "Node metrics", totalMessages
-    discard setTimer(Moment.fromNow(2.seconds), logMetrics)
+
+      let totalErrors = parseCollectorIntoF64(waku_node_errors)
+      let totalConnections = parseCollectorIntoF64(waku_node_conns_initiated)
+
+      # track cumulative, and then max.
+      let freshErrorCount = max(totalErrors - cumulativeErrors, 0)
+      let freshConnCount = max(totalConnections - cumulativeConns, 0)
+      
+      cumulativeErrors = totalErrors
+      cumulativeConns = totalConnections
+      info "Total connections initiated", count = freshConnCount
+      info "Total Messages", count = parseCollectorIntoF64(waku_node_messages)
+      info "Total SWAP peers", count = parseCollectorIntoF64(waku_swap_peers_count)
+      info "Total FILTER peers", count = parseCollectorIntoF64(waku_filter_peers)
+      info "Total errors", count = freshErrorCount
+      info "Total subscribed topics", count = parseCollectorIntoF64(waku_filter_subscribers)
```

Comment:
```suggestion
      info "Total active filter subscriptions", count = parseCollectorIntoF64(waku_filter_subscribers)
```
This counts the number of subscriptions that this node services, not the number of content topics it is subscribed to.

Reply:
@jm-clius, in this [snippet](https://github.com/status-im/nwaku/blob/0deedf0d7ab494d41a09861b44c0e823aa2e0651/waku/v2/node/wakunode2.nim#L243-L271), the number of contentFilters is equivalent to the number of contentTopics. Can we log this out as the total subscribed topics?

Reply:
Indeed! Not sure though if it's necessary to periodically log this, as it's (usually) unlikely to change during the lifetime of a filter client: filter clients most often start with a handful of content topic interests, subscribe to these, and don't really change their subscription over their lifetime. They'd be able to monitor this number using metrics/dashboards, but not sure how valuable it will be to repeat this number every 30s. I think we can consider this in subsequent PRs, but for now the scope of what we log periodically seems 👍 for me. WDYT?

Reply:
Agreed! If the need arises for it, we can definitely log it in the future